### PR TITLE
initial-project-upload

### PR DESCRIPTION
**Pull request by aeh1707**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
```
Comment (aeh1707) [inline index.html:95]: Please remove these unnecessary comments as it may make your code unorganized and hard to maintain.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+        }
+        li {
+          margin-right: 30px;
+        }
+        #nav-bar a, ul li {
+          align-self: center;
+        }
+        
+    }
+</style>
```
Comment (aeh1707) [inline index.html:47]: Please move these styles to an external CSS stylesheet and link it back to your HTML, it's a best practice to have a separation of concerns.

**Review by aeh1707 — COMMENTED** [review]
# Changes Required :recycle: 

## Things you done good:
Great, you're almost there!
:heavy_check_mark: Passed all tests.
:heavy_check_mark: Semantic HTML.

## Things done bad and should improve on:
:x:  Follow the GitHub flow and never push directly on the main, create a branch for each feature.



Please check out my below comments and make the optional changes. :smiley: 
HAPPY CODING :computer:

**Review by aimemalaika — APPROVED** [review]
Hi @aeh1707 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

133591850-fdcb7dd3-dd7a-4274-81e7-0e59434148f7

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**